server/src/service/poleEmploi/offresPoleEmploi.ts

```diff
@@ -15,113 +15,63 @@ import { getAccessToken, getRoundedRadius, peApiHeaders } from "./common.js"
 const blackListedCompanies = ["iscod", "oktogone", "institut europeen f 2i"]
 
 const getSomePeJobs = async ({ romes, insee, radius, lat, long, caller, diploma, opco, opcoUrl, api }) => {
-  // la liste des romes peut être supérieure au maximum de trois autorisés par l'api offre de PE
-  // on segmente les romes en blocs de max 3 et lance autant d'appels parallèles que nécessaires
-  const chunkedRomes = []
-  let i = 0,
-    k = 0
-  while (i < romes.length) {
-    chunkedRomes[k] = romes.slice(i, i + 3).join(",")
-    i += 3
-    ++k
+  let jobs: PEResponse | IApiError = null
+  const currentRadius = radius || 20000
+  const jobLimit = 50 //TODO: query params options or default value from properties -> size || 50
+
+  let trys = 0
+
+  while (trys < 3) {
+    jobs = await getPeJobs({ romes, insee, radius: currentRadius, jobLimit, caller, diploma, api })
+
+    if (jobs.status === 429) {
+      console.log("PE jobs api quota exceeded. Retrying : ", trys + 1)
+      // trois essais pour gérer les 429 quotas exceeded des apis PE.
+      trys++
+      await new Promise((resolve) => setTimeout(resolve, 1000))
```

Comment:
```suggestion
import {
  setTimeout,
} from 'timers/promises';

const res = await setTimeout(100, 'result');
```

[Doc](https://nodejs.org/api/timers.html#timerspromisessettimeoutdelay-value-options)

Reply:
ok adopté.